crates/uv-git/src/credentials.rs

```diff
@@ -30,9 +29,9 @@ impl GitStore {
 /// Populate the global authentication store with credentials on a Git URL, if there are any.
 ///
 /// Returns `true` if the store was updated.
-pub fn store_credentials_from_url(url: &Url) -> bool {
+pub fn store_credentials_from_url(url: &LogSafeUrl) -> bool {
     if let Some(credentials) = Credentials::from_url(url) {
-        trace!("Caching credentials for {}", redacted_url(url));
+        trace!("Caching credentials for {}", url);
```

Comment:
```suggestion
        trace!("Caching credentials for {url}");
```

for `println!` catches those, but apparently it can't through tracing's macros.